native/dotnet-sdk-8.0/digests

```diff
@@ -1,3 +1,3 @@
 dotnet-sdk-8.0.409-linux-x64.tar.gz SHA1 5e004ddaeeb58438e2b7dc916a88086262a295e0
 dotnet-sdk-8.0.409-linux-x64.tar.gz SHA256 0239846ee905f1052c34b80b561231a51dfd5fc7f7588a205627080071ffe4f8
-dotnet-sdk-8.0.409-linux-x64.tar.gz MD5 0e35e71fd521dbdf574f1c0635f95e79
+dotnet-sdk-8.0.409-linux-x64.tar.gz MD5 0e35e71fd521dbdf574f1c0635f95e79
```

Comment:
some as above (use trailing linefeed)
you don't have to manually edit digests files.
Just create those with `make clean && make digests`